test/Worker/Grpc.Tests/Worker.Grpc.Tests.csproj

```diff
@@ -1,11 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net48</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
Same as the above comment to remove net48

Reply:
Same here, the issue is now fixed in `main`